nipype/algorithms/confounds.py

```diff
@@ -464,7 +471,7 @@ def _run_interface(self, runtime):
                              .format(self.inputs.realigned_file, imgseries.ndim, imgseries.shape))
 
         if isdefined(self.inputs.mask_file):
-            in_mask_data = nb.load(self.inputs.mask_file, mmap=NUMPY_MMAP).get_data()
+            in_mask_data = nb.load(self.inputs.mask_file[0], mmap=NUMPY_MMAP).get_data()
```

Comment:
should we consider a union or intersection of masks, and a new Enum trait (for 'union', 'intersect') and xor-ed with an index trait if we want to index just one of the masks.

```
merge_method = traits.Enum('union', 'intersect', 'none', xor=['mask_index'], ...)
mask_index = traits.Range(0, low=0, xor=['merge_method'], ...)
```

`none` corresponds to per mask.